src/controllers/wallet/tx-proposal.controller.js

```diff
@@ -86,9 +86,10 @@ async function signTx(req, res) {
   }
 
   const { txHex } = req.body;
-  const signatures = req.body.signatures || [];
+  const signatures = req.body.signatures.slice(0, req.wallet.multisig.minSignatures) || [];
```

Comment:
Trying to understand the behavior here: assuming `minSignatures` is 2 and there are 5 signatures at maximum.

If a user receives the `txHex` with 2 signatures and runs it through this `signTx` method, it will just add another one, to a total of 3 signatures.
If a user receives the `txHex` with 3 signatures and runs it through `signTx`, it will slice off one of the signatures and add its own, to a total of 3 signatures again.

Is this the desired behavior? I feel like it would want to slice _after_ the `assemblePartialTransaction`, and not before.
If this is not, I believe a test specific for this case would be a good idea.